spec/acceptance/ntp_server_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper_acceptance'
+require 'yaml'
```

Comment:
Do the tests require yaml library?

Reply:
Then it is not worth adding it either to all tests or to spec_helper_acceptance?

Reply:
Ah my bad, didn't see the file thought this was the spec_helper_acceptance - good spot @willmeek 👍